interfaces/builtin/host_docs_test.go

```diff
@@ -0,0 +1,118 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin_test
+
+import (
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/interfaces/builtin"
+	"github.com/snapcore/snapd/interfaces/mount"
+	"github.com/snapcore/snapd/snap"
+	"github.com/snapcore/snapd/testutil"
+)
+
+type hostDocsSuite struct {
+	iface        interfaces.Interface
+	coreSlotInfo *snap.SlotInfo
+	coreSlot     *interfaces.ConnectedSlot
+	plugInfo     *snap.PlugInfo
+	plug         *interfaces.ConnectedPlug
+}
+
+var _ = Suite(&hostDocsSuite{iface: builtin.MustInterface("host-docs")})
+
+const hostDocsConsumerYaml = `name: consumer
+version: 0
+apps:
+ app:
+  plugs: [host-docs]
+`
+
+const hostDocsCoreYaml = `name: core
+version: 0
+type: os
+slots:
+  host-docs:
+`
+
+func (s *hostDocsSuite) SetUpTest(c *C) {
```

Comment:
Should this test `MockOnClassic(true)`?

Reply:
I guess technically yes, though we never did that on any of the interface tests.

Reply:
We do have this in ~12 interfaces, e.g. x11_test.go, udisks2_test.go.

Reply:
Done